### ISSUE TITLE
v4.3.0 has introduced a breaking change to our unit tests

### ISSUE THREAD

**Issue by alexross1988** [react-dropzone/react-dropzone#638]
**Do you want to request a *feature* or report a *bug*?**

- [x] I found a bug
- [ ] I want to propose a feature

**What is the current behavior?**
Before release 4.3.0 we had a test that did:
```
    // Create a non-null file
    const fileContents = 'file contents'
    const file = new Blob([fileContents], { type: 'text/plain' })

    wrapper.find('.FileDrop__Dropzone').at(0).simulate('drop', { dataTransfer: { files: [file] } })
    expect(wrapper.state('files')).toEqual([file])
    expect(onFileDrop).toBeCalled()
```

This passed, but now we get:
```
  ● FileDrop › calls onFileDrop when a file is dropped

    expect(received).toEqual(expected)

    Expected value to equal:
      [{Symbol(impl): {"_buffer": {"data": [102, 105, 108, 101, 32, 99, 111, 110, 116, 101, 110, 116, 115], "type": "Buffer"}, "type": "text/plain", Symbol(wrapper): [Circular]}}]
    Received:
      []
```

Pinning back to v4.2.13 fixes our test

**If the current behavior is a bug, please provide the steps to reproduce.**

1.
1.
1.

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Operating System and react-dropzone version.**

**Comment by asfordmatt** [comment]
Related to this PR: https://github.com/react-dropzone/react-dropzone/pull/616

We've been able to get our tests passing by adding async/await similar to the above PR

**Comment by okonet** [comment]
Hmm, probably we should have released as a breaking change indeed.

**Comment by okonet** [comment]
Does anyone wants to update the documentation to mention that?

**Comment by tobilen** [comment]
https://github.com/react-dropzone/react-dropzone/pull/645/commits/0651264c87eb83fe64ec68ec47e130058fb1fca1 fixes the issue

**Comment by okonet** [comment]
So can this be closed?

**Comment by okonet** [comment]
Should we deprecate the release and release as a major to remove confusion?

**Comment by tobilen** [comment]
solved by #645 

[already did a couple comments ago though](https://github.com/react-dropzone/react-dropzone/issues/638#issuecomment-414864139). you also get a specific close message here when merging a PR with the `closes` or `fixes` keyword, indicating that it was solved via this PR 🤓
